Sources_v3/StreamChatUI/ChatChannel/ChatMessageErrorIndicator.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageErrorIndicator<ExtraData: ExtraDataTypes>: Button, UIConfigProvider {
+    override public func defaultAppearance() {
+        setImage(UIImage(named: "error_indicator", in: .streamChatUI)!, for: .normal)
+        tintColor = uiConfig.colorPalette.outgoingMessageErrorIndicatorTint
```

Comment:
nit: `defaultAppearance` should call super

Reply:
added in e34ad267831814602d3c307bacef932ba04ae167